vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/permission_rules/full_screen_mode_frequency_cap_unittest.cc

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/permission_rules/full_screen_mode_frequency_cap.h"
+
+#include "bat/ads/internal/unittest_base.h"
+#include "bat/ads/internal/unittest_util.h"
+
+// npm run test -- brave_unit_tests --filter=BatAds*
+
+namespace ads {
+
+class BatAdsFullScreenModeFrequencyCapTest : public UnitTestBase {
+ protected:
+  BatAdsFullScreenModeFrequencyCapTest() = default;
+
+  ~BatAdsFullScreenModeFrequencyCapTest() override = default;
+};
+
+TEST_F(BatAdsFullScreenModeFrequencyCapTest, AllowAd) {
+  // Arrange
+
+  // Act
+  MockFullScreenMode(ads_client_mock_, false);
+
+  // Assert
+  FullScreenModeFrequencyCap frequency_cap;
+  const bool is_allowed = frequency_cap.ShouldAllow();
```

Comment:
nit: in other frequency cap tests we have `ShouldAllow()` under `// Act` (and `MockFullScreenMode(..)` would be under `// Arrange` respectively

Reply:
same for all cases in this test

Reply:
Fixed